alacritty_terminal/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to alacritty_terminal are documented in this file. The
+sections should follow the order `Added`, `Changed`, `Deprecated`, `Fixed` and
+`Removed`.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## 0.23.0-dev
+
+### Changed
+
+- `Term<T>` is not focused by default anymore
```

Comment:
```suggestion
- `Term` is not focused by default anymore
```

The generics are just unnecessary noise. Providing a full path could be nice, but shouldn't be necessary in this case.